scripts/govcms-deploy

```diff
@@ -6,9 +6,6 @@ set -euo pipefail
 # Determine the config strategy.
 GOVCMS_CONFIG_STRATEGY=${GOVCMS_CONFIG_STRATEGY:-import}
 
-# Ensure placeholders are replaced in Drush9 alias file.
-#sed -i "s/%%PROJECT_NAME%%/$LAGOON_PROJECT/g" /app/drush/sites/govcms.site.yml
```

Comment:
This has moved to a drush alias that lives at /etc/drush/sites/ci.site.yml - placed via Dockerfile. The major benefit of this location is that we reduce the number of files that can be WIPED by docker-compose mounting /app.